lnbits/core/views/api.py

```diff
@@ -621,29 +621,20 @@ async def api_lnurlscan(code: str, wallet: WalletTypeInfo = Depends(get_key_type
 
 
 @core_app.post("/api/v1/payments/decode", status_code=HTTPStatus.OK)
-async def api_payments_decode(data: DecodePayment, response: Response):
+async def api_payments_decode(data: DecodePayment) -> JSONResponse:
     payment_str = data.data
     try:
         if payment_str[:5] == "LNURL":
             url = lnurl.decode(payment_str)
-            return {"domain": url}
+            return JSONResponse({"domain": url})
         else:
             invoice = bolt11.decode(payment_str)
-            return {
-                "payment_hash": invoice.payment_hash,
-                "amount_msat": invoice.amount_msat,
-                "description": invoice.description,
-                "description_hash": invoice.description_hash,
-                "payee": invoice.payee,
-                "date": invoice.date,
-                "expiry": invoice.expiry,
-                "secret": invoice.secret,
-                "route_hints": invoice.route_hints,
-                "min_final_cltv_expiry": invoice.min_final_cltv_expiry,
-            }
```

Comment:
This is a breaking API change. Above is the new version and underneath latest main
![image](https://github.com/lnbits/lnbits/assets/8138838/a901099e-40d4-482c-b2c7-5d2e3f6f4b31)

Shouldn't cause any major issues with backwards compatibility though

Funnily enough, the main version decodes the `min_final_cltv_expiry` incorrectly

Reply:
hm i dont see a lot of problem, there are only additional fields, but the ones that were there before are there also

Reply:
> hm i dont see the problem, there are only additional fields [...]

As far as I can see, `secret` changed to `payment_secret` and `route_hints` is missing - this will easily break existing clients.

Reply:
@dni why is this marked as resolved?